core/runtime/ops.rs

```diff
@@ -343,6 +343,33 @@ pub fn serde_v8_to_rust<'a, T: Deserialize<'a>>(
   from_v8(scope, input)
 }
 
+/// Retrieve a [`serde_v8::V8Slice`] from a value.
+#[allow(clippy::result_unit_err)]
+pub fn to_nonresizable_v8_slice(
+  scope: &mut v8::HandleScope,
+  input: v8::Local<v8::Value>,
+) -> Result<serde_v8::V8Slice, ()> {
```

Comment:
Is there a specific reason why this function returns a unit value for the error?

Reply:
This code is a bit awkward right now -- the error is specified in the generated code and this just returns a success or failure. It's somewhat complicated to deal with errors in the codegen parts and it matches the v8 casts above.

I would eventually like to have these return &' static str that becomes the actual JS error.

I'll see if there's something I can do here right now to switch away from the unit errors, but I may need to do a pass on all the op args to ensure that errors work as we expect.

Reply:
Fixed in the latest -- we now use a static string here. I'll add a TODO on the roadmap to circle back on errors, but it should be good enough for now I think.